bundles/org.openhab.binding.insteon/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -137,6 +142,11 @@
 		<label>Cool Set Point</label>
 	</channel-type>
 
+	<channel-type id="dawndusk">
```

Comment:
Can we use something more generic than "dawndusk"? This assumes that the variable is associated with the sun rise/set.

Reply:
change to:

	<channel-type id="lightdark">
		<item-type>Contact</item-type>
		<label>Light/Dark</label>
	</channel-type>